new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodsShutdownOption.java

```diff
@@ -0,0 +1,291 @@
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static final String managedServer1Name = "managed-server1";
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain();
+  }
+
+  /**
+   * This test is to verify different shutdown options specified at different scopes in Domain Resource Definition.
+   * Refer to section "Shutdown options" of Documentation link:
+   * https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/
+   * step 1: Startup a WebLogic domain with one cluster that initially has one running managed server. The shutdown option is
+   * configured as follows:
+   * domain: SHUTDOWN_TYPE -> Graceful.
+   * adminServer: SHUTDOWN_TYPE -> Forced.
+   *              SHUTDOWN_IGNORE_SESSIONS -> true.
+   *              SHUTDOWN_TIMEOUT -> 40.
+   * cluster: SHUTDOWN_IGNORE_SESSIONS -> true.
+   *          SHUTDOWN_TIMEOUT -> 80.
+   * managedServer1: SHUTDOWN_TIMEOUT -> 100.
+   * step2: Scale cluster with two managed servers.
+   * step 3: Verify shutdown properties of admin server, managedServer1 and newly scaled up managedServer2.
+   * Domain level "Graceful" SHUTDOWN_TYPE overrides server level setting and is used for all servers.
+   * So adminServer has "Graceful" SHUTDOWN_TYPE, default "false" SHUTDOWN_IGNORE_SESSIONS and
+   * configured "40" SHUTDOWN_TIMEOUT.
+   * Managed level setting overrides cluster level setting so managedServer1 has configured "100"
+   * SHUTDOWN_TIMEOUT, cluster level "true" SHUTDOWN_IGNORE_SESSIONS  "true" and "Graceful" SHUTDOWN_TYPE.
+   * Newly scaled up managedServer2 takes setting from combination of domain level and cluster level
+   * with "Graceful" SHUTDOWN_TYPE, "true" SHUTDOWN_IGNORE_SESSIONS and default "30" SHUTDOWN_TIMEOUT
+   */
+  @Test
+  @DisplayName("Verify shutdown rules when shutdown properties are defined at different levels ")
+  public void testShutdownProps() throws ApiException {
+
+    //scare the cluster to 2 managed servers
```

Comment:
scare the cluster?  It is not Halloween yet...  Did you mean "scale"?

Reply:
Corrected. Thanks!